extensions/container-image/container-image-openshift/deployment/src/main/java/io/quarkus/container/image/openshift/deployment/OpenshiftConfig.java

```diff
@@ -113,6 +115,12 @@ public static String getDefaultJvmImage(CompiledJavaVersionBuildItem.JavaVersion
     @ConfigItem(defaultValue = "PT5M")
     Duration buildTimeout;
 
+    /**
+     * The log level of OpenShift build log.
+     */
+    @ConfigItem(defaultValue = DEFAULT_BUILD_LOG_LEVEL)
+    public String buildLogLevel;
```

Comment:
Why not using directly the enum type? I mean, use Logger.Level instead of String.

Reply:
Done. Thank you